server/src/test/java/org/elasticsearch/snapshots/SnapshotResiliencyTests.java

```diff
@@ -397,9 +401,22 @@ public void testSnapshotWithNodeDisconnects() {
             } else if (randomBoolean()) {
                 scheduleNow(() -> testClusterNodes.clearNetworkDisruptions());
             }
+        }, e -> {
+            if (partial == false) {
+                final Throwable unwrapped =
```

Comment:
final SnapshotException ... and then assertNotNull() in the next line

Reply:
Done :)